packages/store-sync/src/postgres/createStorageAdapter.ts

```diff
@@ -0,0 +1,218 @@
+import { PublicClient, encodePacked, size } from "viem";
+import { PgDatabase, QueryResultHKT } from "drizzle-orm/pg-core";
+import { and, eq } from "drizzle-orm";
+import { StoreConfig } from "@latticexyz/store";
+import { debug } from "./debug";
+import { tables } from "./tables";
+import { spliceHex } from "@latticexyz/common";
+import { setupTables } from "./setupTables";
+import { StorageAdapter, StorageAdapterBlock } from "../common";
+
+// Currently assumes one DB per chain ID
```

Comment:
is this comment still accurate?

Reply:
yes

Reply:
I was confused about the `chain` table but I now understand the expectation is to have only one row in that table, and we don't have a `chainId` column in the data table so couldn't support multiple chains in the same data table with the current schema